cherrypy/_cptree.py

```diff
@@ -217,6 +217,11 @@ def mount(self, root, script_name='', config=None):
                 'request). You cannot mount such Applications on this Tree; '
                 'you must pass them to a WSGI server interface directly.')
 
+        if script_name and not script_name.startswith('/'):
+            cherrypy.log.error(
+                msg='script_name {0} should start with /'.format(script_name)
+            )
+            script_name = '/' + script_name
```

Comment:
I'm disinclined to continue to let the user supply an erroneous value and correct it for them. Why not instead hard-fail (raise an exception) when invalid values are input (similar to how `script_name=None` is handled)?

Reply:
fixes the script_name to prevent 404 error (not sure if it's necessary) , its what i said in my first comment. 
i don't think letting user to continue is a right solution. but in doc i saw this:
This should start with a slash

so i choose to fix error myself and throw a console error.
raise an exception is indeed a better solution